fhir-bucket/src/main/resources/stats/postgres/str_values.sql

```diff
@@ -0,0 +1,10 @@
+select * from fhirdata.observation_str_values 
```

Comment:
this feels like debug code, and probably not useful here

Reply:
A lot of these queries are there in case we need to refer to them again in the future as they are useful when investigating issues. They're not really test code either, which is why I opted to stick them under src/main.